lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+module ActiveAdmin
+  module Middleware
+    class ErrorHandler
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        @app.call(env)
+      rescue Exception => e
+        if ActiveAdmin.application.default_namespace.to_s == env['PATH_INFO'].split('/').second
```

Comment:
@timoschilling I feel the way that I'm checking if the request-response is going thur ActiveAdmin (and handling the exceptions)  is kind of not so great. I would like your thoughts here. What can I change/improve ?